.github/workflows/ubuntu-bundler3.yml

```
@@ -0,0 +1,44 @@
+name: ubuntu-bundler3
+
+on:
+  pull_request:
+
+  push:
+    branches:
+      - staging
+      - trying
+
+jobs:
+  ubuntu-bundler3:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ 2.4.9, 2.5.7, 2.6.5 ]
+        rgv: [ v3.0.6, master ]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - name: Prepare dependencies
+        run: |
+          bin/rake override_version
```

This is the step that needs `BUNDLER_SPEC_SUB_VERSION`. This rake task replaces the content of `lib/bundler/version.rb` with what's in `BUNDLER_SPEC_SUB_VERSION` so that the following steps can run in "bundler 3 mode". So:

```suggestion
          BUNDLER_SPEC_SUB_VERSION=3.0.0 bin/rake override_version
```